auto_round/mllm/README.md

```diff
@@ -1,57 +1,52 @@
 # AutoRound for MLLMs
+ This feature is experimental and may be subject to changes, including potential bug fixes, API modifications, or adjustments to default parameters
 
 ## Quantization
 
 ### API Usage (Gaudi2/CPU/GPU) Recommended
-AutoRound uses the text module of MLLM (LLM component) as the main quantization target. with NeelNanda/pile-10k as the default calibration dataset.
-
+In default, AutoRoundMLLM only quantize the text module of VLMs and use NeelNanda/pile-10k" for calibration
 ```python
     from auto_round import AutoRoundMLLM
     from transformers import Qwen2VLForConditionalGeneration, AutoProcessor, AutoTokenizer
+    
     ## load the model
     model_name = "Qwen/Qwen2-VL-2B-Instruct"
-    tokenizer = AutoTokenizer.from_pretrained(model_name)
-    processor = AutoProcessor.from_pretrained(model_name, trust_remote_code=trust_remote_code)
     model = Qwen2VLForConditionalGeneration.from_pretrained(
-        model_name, trust_remote_code=True)
-        
+        model_name, trust_remote_code=True)  
+    tokenizer = AutoTokenizer.from_pretrained(model_name)
+    processor = AutoProcessor.from_pretrained(model_name, trust_remote_code=True)
+
     ## quantize the model
-    bits, group_size = 4, 128
-    autoround = AutoRoundMLLM(model, tokenizer, processor, bits=bits, group_size=group_size)
+    bits, group_size, sym = 4, 128, True
+    autoround = AutoRoundMLLM(model, tokenizer, processor, bits=bits, group_size=group_size, sym=True)
     autoround.quantize()
 
     # save the quantized model, set format='auto_gptq' to use AutoGPTQ format
     output_dir = "./tmp_autoround"
     autoround.save_quantized(output_dir, format='auto_round', inplace=True)
 ```
 
-- `dataset`: the dataset for quantization training. current support NeelNanda/pile-10k,llava_conv_58k,llava_instruct_80k and llava_instruct_150k. It can be a custom one. Please note that the effectiveness of the Llava calibration dataset has only been validated on five models so far.
+- `dataset`: the dataset for quantization training. current support NeelNanda/pile-10k,llava_conv_58k,llava_instruct_80k and llava_instruct_150k. Please note that the effectiveness of the Llava calibration dataset has only been validated on five models so far.
```

Comment:
please notices the blank space

NeelNanda/pile-10k, llava_conv_58k, llava_instruct_80k